### PR TITLE
Add tensor FFs HQE bounds and new B->D* LQCD constraints

### PR DESCRIPTION
**Pull request by gubernari**
- [x] Add BGL tensor bounds as function of the IW functions
- [x] Add FNAL/MILC, HPQCD, and JLQCD B->D* constraints

### CODE REVIEW

File: eos/parameters/hadronic-matrix-elements.yaml [hunk 1]
```diff
@@ -761,6 +761,29 @@ groups:
...
+          matrix:
+            - ["f+", "f0", "fT"]
+            - [0, 1, 2, 3, 4]
+      'B->D::tp@BFW2010' :
+          central:  51.1082
+          min:      51.1082
+          max:      51.1082
+          latex:    '$t_+^{B\to D}$'
+          comment:  '(mB + mD)^2'
+          unit:     'GeV'
```
Comment (mreboud) [inline eos/parameters/hadronic-matrix-elements.yaml:780]: The unit is incorrect

**Review by mreboud — COMMENTED** [review]
I left comments :)

**Comment by dvandyk** [comment]
@gubernari Can you please split that commit into one fixup commit for the whitespaces and one clean commit adding the new constraint?

File: eos/form-factors/observables.cc [hunk 2]
```diff
@@ -1843,6 +1843,14 @@ namespace eos
                         Unit::None(),
                         &HQETUnitarityBounds::bound_1m),
 
+                make_observable("b->c::Bound[1^+,T]@CLN", R"(B^{b\to c}_{1^+,T})",
+                        Unit::None(),
+                        &HQETUnitarityBounds::bound_1p_T),
+
+                make_observable("b->c::Bound[1^-,T]@CLN", R"(B^{b\to c}_{1^-,T})",
+                        Unit::None(),
+                        &HQETUnitarityBounds::bound_1m_T),
```
[the remark this answers is not in the document]
Comment (gubernari) [reply eos/form-factors/observables.cc:1887]: It is a different notation, but I believe is consistent (the parity is indicated by a "+" or a "-")

[the remark this answers is not in the document]
Comment (gubernari) [reply eos/form-factors/observables.cc:1887]: I will simply remove the BFW2010 commit for the moment being

[the remark this answers is not in the document]
Comment (gubernari) [reply eos/form-factors/observables.cc:1887]: I talked to Danny and he is fine with that notation

File: eos/parameters/hadronic-matrix-elements.yaml [hunk 3]
```diff
@@ -1915,6 +1961,37 @@ groups:
...
+            - [0, 1, 2, 3, 4]
+      'B->D^*::tp_a@BFW2010' :
+          central:  53.1339
+          min:      53.1339
+          max:      53.1339
+          latex:    '$t_+^{V, B\to D^*}$'
+          comment:  '(mB + mD*)^2'
+          unit:     'GeV'
+      'B->D^*::tp_v@BFW2010' :
+          central:  53.1339
```
[the remark this answers is not in the document]
Comment (gubernari) [reply eos/parameters/hadronic-matrix-elements.yaml]: I will simply remove the BFW2010 commit for the moment being

File: eos/parameters/hadronic-matrix-elements.yaml (hunk 1 above)
Comment (gubernari) [reply eos/parameters/hadronic-matrix-elements.yaml]: I will simply remove the BFW2010 commit for the moment being

**Comment by gubernari** [comment]
> @gubernari Can you please split that commit into one fixup commit for the whitespaces and one clean commit adding the new constraint?

Please have a look now!

File: eos/form-factors/observables.cc (hunk 2 above)
[the remark this answers is not in the document]
Comment (mreboud) [reply eos/form-factors/observables.cc:1887]: > I talked to Danny and he is fine with that notation

I should probably stop reviewing then ><

[the remark this answers is not in the document]
Comment (dvandyk) [reply eos/form-factors/observables.cc:1887]: I agree that the notation is weird, but I would keep it for this / now.
@gubernari can you open an issue reminding us to harmonize the notation of the various unitarity bounds across EOS in the future? This issue should also address that here we use ``Bound`` for the observable, while everywhere else we use ``Saturation``. The ``Bound`` refers to the constraint only for all other unitarity-bounded parametrisations.

**Comment by dvandyk** [comment]
> > @gubernari Can you please split that commit into one fixup commit for the whitespaces and one clean commit adding the new constraint?
> 
> @dvandyk Please have a look now!

Will review. Keep the new issue in mind!

**Review by dvandyk — APPROVED** [review]

File: eos/form-factors/observables.cc (hunk 2 above)
[the remark this answers is not in the document]
Comment (mreboud) [reply eos/form-factors/observables.cc:1887]: I did it

**Pull request merged**